app/scripts/metamask-controller.js

```diff
@@ -4938,8 +4961,6 @@ export default class MetamaskController extends EventEmitter {
         ),
         getProviderConfig: () => this.networkController.state.providerConfig,
         setProviderType: (type) => {
-          // when using this format, type happens to be the same as the networkClientId...
-          this.selectedNetworkController.setNetworkClientIdForMetamask(type);
```

Comment:
this is no longer necessary because `SelectedNetworkController` no longer keeps track of the globally `selectedNetworkClientId`